frontend/src/scenes/pipeline/pipelineDefaultEnabledLogic.tsx

```diff
@@ -0,0 +1,55 @@
+import { connect, kea, path, selectors } from 'kea'
+
+import type { pipelineDefaultEnabledLogicType } from './pipelineDefaultEnabledLogicType'
+import { pipelineTransformationsLogic } from './transformationsLogic'
+
+interface PluginContent {
+    title: string
+    description: string
+}
+type PluginContentMapping = Record<string, PluginContent>
+const pluginContentMapping: PluginContentMapping = {
+    GeoIP: {
```

Comment:
I would actually like this to be in the api where-ever we choose to add geoIP plugin for everyone, but this was the fast way